src/app/design/guidelines/back-button/index.html

```diff
@@ -0,0 +1,71 @@
+<stache
+  pageTitle="Back Button"
+  navTitle="Back button"
+  showTableOfContents="true">
+
+  <stache-page-anchor>
+    New page behavior
+  </stache-page-anchor>
+
+  <p>
+    When users navigate to a new context such as a new record, list or section, the action should create an entry in the browser history with a new page title. This entry enables users to select the browser's back button to return to the previous page or view.
+  </p>
+  <p>
+    As a rule of thumb, create a new page in the browser history if the majority of the page content changes or users perceive a signficant change of context. Don't create a new URL or new browser entry for an inline interaction such as steps in a wizard or opening a detail panel.
+  </p>
+
+  <div class="sky-subsection-heading">Focus</div>
+
+  <p>
+    After loading a new page, users should be able to use the tab key to reach the first focusable element on the page.
+  </p>
+
+  <div class="sky-subsection-heading">Screen readers</div>
+
+  <p>
+    When a page loads, it should behave similarly to what users expect from a regular server-side application. The page should announce the page's <sky-code>&lt;title&gt;</sky-code> to the screen reader and inform users that it loaded via an ARIA live region when the <sky-code>&lt;title&gt;</sky-code> element is ready. Behaviors vary based on different browser and screen reader combinations, but SKY UX applications use this combination.
+  </p>
+
+
+  <div class="sky-subsection-heading">Focus</div>
+
+  <stache-page-anchor>
+    Maintaining context
+  </stache-page-anchor>
+
+  <p>
+    For inline interactions, determine the persistence level that makes sense for the change of state that user initiates. When users select the back button, return them to the context they left. Don't make them re-orient, and in some cases re-interact with the system, to return to that context.
```

Comment:
I'd change "user initiates" to "users initiate"

I wonder if we can tighten up the last sentence to "Don't make them re-orient or re-interact with the system to return to that context."